packages/flutter/lib/src/animation/animations.dart

```diff
@@ -607,6 +607,24 @@ abstract class CompoundAnimation<T> extends Animation<T>
   }
 }
 
+/// An animation of [double]s that tracks the sum of two given animations.
```

Comment:
looks like you don't need this class after all so you can remove it and get away without changing the example. :-)